tests/cdkproxy/test_environment_stack.py

```diff
@@ -5,6 +5,17 @@
 
 from dataall.cdkproxy.stacks import EnvironmentSetup
 
+@pytest.fixture(scope='function', autouse=True)
+def patch_methods_sagemaker_studio_extension(mocker):
+    # TODO = WAYS OF TESTING EXTENSIONS outside of environment testing
```

Comment:
I believe we should do it this way:
in `test_environment_stack`, we should have a test that covers creation of environment stack whether some extension are enabled of not. Basically, it's just a test of an environment stack without extensions.
In `test_mlstudio_environment_stack`, we should test only extension of mlstudio. We can omit testing of base environment stack, since it was tested in `test_environment_stack`.

It's needed for this: If we turn off the ML studio module we shouldn't not test the extension since it won't be there. `test_mlstudio_environment_stack` will be in the module folder and will be enabled/disabled together with the module

Reply:
@dbalintx can we merge this PR? I will work on the testing in a separate PR. I fully agree with Nikita's approach on it

Reply:
its fine from my side to merge, if we have the testing in a followup.
Can we also create a note somewhere to cleanup the mlstudio/notebook permissions+ service policies? just so that we won't forget it at the end of the modularization.